### DISCUSSION TITLE
New Source: SingleStore

### DISCUSSION THREAD

**Discussion by lamnguyen92dn** [airbytehq/airbyte#35871] [New Connector Request]
I need SingleStore Connector.
Tks

**Comment by oyeliseiev-ua** [comment]
Working on this in PR https://github.com/airbytehq/airbyte/pull/37337

**Reply by mhmaref** [reply]
Hello,
I don't find SingleStore as a destination DB in the self-managed Airbyte installed on Kubernetes, is it because the connector is still under development? what is the current work around? use MySQL connector?

**Reply by oyeliseiev-ua** [reply]
Hi, 
Plan to start development soon, as for now you can use not published python connector: https://github.com/mohamagdy/airbyte/pull/2/files.

[a comment by an automated account is not shown]
